mmv1/third_party/terraform/tests/resource_compute_instance_template_test.go.erb

```diff
@@ -173,26 +173,45 @@ func TestComputeInstanceTemplate_scratchDiskSizeCustomizeDiff(t *testing.T) {
 		Typee       string // misspelled on purpose, type is a special symbol
 		DiskType    string
 		DiskSize    int
+		Interfacee  string
 		ExpectError bool
 	}{
-		"scratch disk correct size": {
+		"scratch disk correct size 1": {
 			Typee:       "SCRATCH",
 			DiskType:    "local-ssd",
 			DiskSize:    375,
+			Interfacee:  "NVME",
+			ExpectError: false,
+		},
+		"scratch disk correct size 2": {
+			Typee:       "SCRATCH",
+			DiskType:    "local-ssd",
+			DiskSize:    375,
+			Interfacee:  "NVME",
 			ExpectError: false,
 		},
```

Comment:
what's the difference these two test cases?

Reply:
Good catch! One of those definitely said `DiskSize: 3000` at some point. Fixed.

Tests have also been pushed.